gameObjects.js

```diff
@@ -76,13 +83,19 @@ function setData() {
 
 //Map could contain 128x128 celestialPoint() objects
 var gameSpace = {
-  map : [spaceship.maxCoord + 1][spaceship.maxCoord + 1],
-}
+  map : [spaceship.maxCoord + 1][spaceship.maxCoord + 1]
+};
 
 function celestialPoint() {
 
   this.planet = false;
   this.asteroid = false;
 	
 	//Incomplete object constructor for celestial points
+}
+
+//Listener for 'Proceed' button
+document.getElementById("proceed").onclick = function()
```

Comment:
I might be wrong but this could be shortened to
```
document.getElementById("proceed").onclick = spaceship.move;
```

Reply:
For some reason that messes up the energy and supply value decreasing within the move function. The coordinates still update, but other stuff seems to break with that syntax change.

Reply:
Huh, I wonder why.

Reply:
I have a feeling this might be a bigger bug. Wanna open an issue on this and leave for later?